packages/layout/src/lib/header/README.md

```diff
@@ -14,9 +18,20 @@ Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to
 
 ### API
 
+#### Component
+
 | Name         | Default value | Description |
 | -----------  | ------ | -------------------------- |
-| `@Input() your: string;` | `'API'` | Goes here. |
+| `@Input() hasLogo: Boolean;` | `false` | Whether there is a logo or not in the header. Also sets the `has-logo` class. |
+| `@Input() hasContent: Boolean;` | `false` | Whether there is content or not in the header. |
+
+#### Available directives
+
+- auiHeaderContent
```

Comment:
Misschien wat toelichting geven waar je deze gebruikt? Of verwachten we dat de gebruikers dat oppikken uit de examples?